assets/src/edit-story/app/story/useStoryReducer/test/arrangeElement.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Internal dependencies
+ */
+import { LAYER_DIRECTIONS } from '../../../../constants';
+import { setupReducer } from './_utils';
+
+describe( 'arrangeElement', () => {
+	it( 'should move element to specified position', () => {
```

Comment:
Could we add a test with an illegal position?

Reply:
Done in 757a64a5.